flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/model/node/ResourceActionCallBuilder.java

```diff
@@ -68,17 +71,45 @@ public Map<Path, List<TextEdit>> toSource(SourceBuilder sourceBuilder) {
         if (connection.isEmpty()) {
             throw new IllegalStateException("Client must be defined for an action call node");
         }
+
+        Codedata codedata = flowNode.codedata();
+
+        Set<String> ignoredKeys = new HashSet<>(List.of(Property.CONNECTION_KEY, Property.VARIABLE_KEY,
+                Property.TYPE_KEY, TARGET_TYPE_KEY, Property.RESOURCE_PATH_KEY,
+                Property.CHECK_ERROR_KEY));
+        String resourcePath;
+
+        if (codedata.org().equals("ballerina") && codedata.module().equals("http")) {
```

Comment:
Why can't we apply this to any resource path that contains only path parameters?